### PR TITLE
remove image presenter

### PR DESCRIPTION
**Pull request by Arieldelossantos**
GitHub Issue: #

## Proposed Changes

 - Refactoring (no functional changes, no api changes) 

## Checklist

Please check if your PR fulfills the following requirements:

- [x] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [x] Contains **NO** breaking changes
- [x] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

**Comment by DerekRaven** [comment]
Don't forget any .md files that may exist for controls being deleted, they need to be deleted as well.

On that note would it be valuable to also provide info directly in this repo of which controls are being deleted and what they could be replaced with? I'm not sure if we should create a completely new document just for tracking the deletions and the alternative that can be used in other libraries, or at the very least adding a note in the changelog along with the removal.

@MatFillion Any input on this? Should we create some sort of ``MigrationPath.md`` or ``Cleanup-2022.md`` that tracks this?

**Comment by MatFillion** [comment]
@DerekRaven I like the idea of a Migration.md or we simply note them in Changelog.md. doesn't matter if we link to umbrella controls.

**Comment by Arieldelossantos** [comment]
Changelog file modified @DerekRaven @MatFillion

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,15 +6,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
-### Fixed
+### **Fixed**
 
-### Security
+### **Security**
+
+### **Altenative to use**
+| **Removed Control** | **Alternative**|
+|------|-----------------|
```
Comment (DerekRaven) [inline CHANGELOG.md]: Wouldn't it make sense to just have this table in the Removed section?

**Review by DerekRaven — APPROVED** [review]

**Review by DanielCauser — APPROVED** [review]

**Pull request merged**